s3same/iam.py

```diff
@@ -0,0 +1,75 @@
+from botocore.exceptions import ClientError
+
+IAMName = 's3same_travis'
+
+def _policy_string(bucket):
+    return """{
+      "Version": "2012-10-17",
+      "Statement": [
+        {
+          "Action": [
+            "s3:ListBucket"
+          ],
+          "Effect": "Allow",
+          "Resource": [
+            "arn:aws:s3:::%(bucket)s"
+          ]
+        },
+        {
+          "Action": [
+            "s3:PutObject",
+            "s3:PutObjectAcl"
+          ],
+          "Effect": "Allow",
+          "Resource": [
+            "arn:aws:s3:::%(bucket)s/*"
+          ]
+        }
+      ]
+    }""" % dict(bucket=bucket)
```

Comment:
Though now having looked through the rest of the PR, it looks like avoiding having to escape those braces may have been your intent.